### ISSUE TITLE
[2.0.17] Massive amount of duplicate database queries for template_styles

### ISSUE THREAD

**Issue by codeling** [JoomShaper/helix-ultimate#379]
### Steps to reproduce the issue

Use helixultimate template, set site to debug.

### Expected result

Limited or no amount of duplicate database queries.

### Actual result

The following query is executed 117 times:
```
SELECT *
FROM `#__template_styles`
WHERE `client_id` = 0 AND `id` = '9'
```
(client_id and id are probably specific to my setup, but you get the gist)

### System information (as much as possible)

Joomla 4.4.0. apache 2.4.57 on Ubuntu 23.10.

**Comment by joomleb** [comment]
@codeling 
Please, Can you replicate the issue with last Joomla 4.4.5 + Helix Ultimate 2.1.1 ?
Which version of PHP are you running?

**Comment by codeling** [comment]
In my current attempt on the starting page of our website I see 112 reports of the duplicate query as shown in my initial report, so yes, I can still replicate this.

Is there anything else I can do to help here? I have little knowledge with how the Joomla templating system works or how helix-ultimate uses it, so I don't really know where I could look for more information. I do have php development experience so with some guidance I might be able to find out more about whether this is something unique to our setup or a more general thing.

**Comment by joomleb** [comment]
@mi-prakash @ahamed 
Please, Can you help here at least giving to @codeling (thank you) the right instructions to help all of us ?

**Comment by Sieger66** [comment]
Everyone can test a temporary fix from there:

https://forum.joomla.de/thread/21364-helix-template-1-000-mal-die-gleichen-queries/?postID=158848#post158848

for the file:
/plugins/system/helixultimate/src/Platform/Helper.php

**Comment by joomleb** [comment]
@Sieger66 
Thank you for helping. To avoid mistakes, Please, Can you add here the modded helper.php file to replace ?

@mi-prakash 
Please, Do you have any news about this fixing ?

**Comment by joomleb** [comment]
@mi-prakash @ahamed 
Please, Can you confirm the fixing adding as suggested by @Sieger66 ?

**Comment by HDsports** [comment]
The fix was made more than two months ago and Joomshaper doesn't even bother to respond. Wow, terrible support :(

**Comment by joomleb** [comment]
@mi-prakash @ahamed 
Please, Is Helix Ultimate an abandoned project?

**Comment by HDsports** [comment]
I don't think so. Joomsharp support is simply miserable. Helix is the free template from Joomsharp. The developers don't even seem to think it's worth investing a few minutes of their time to fix the error. This is very shameful for such a company.

**Comment by Sieger66** [comment]
> [@Sieger66](https://github.com/Sieger66) Thank you for helping. To avoid mistakes, Please, Can you add here the modded helper.php file to replace ?

See my PR#415 with the file at: 
https://github.com/JoomShaper/helix-ultimate/pull/415

https://raw.githubusercontent.com/JoomShaper/helix-ultimate/e396f34ea53be1dd85a74d167b9a628612ca64ea/plugins/system/helixultimate/src/Platform/Helper.php

**Comment by joomleb** [comment]
Hi guys,
Helix Ultimate 2.1.2 has been released 
@mi-prakash @ahamed
Please, Has this fix been included to the 2.1.2 release ?

**Comment by HDsports** [comment]
No, they didn't have the fix integrated. Joomshaper you should be ashamed of yourselves. You ignore all messages and are even too lazy to implement a fix that costs you a minute of your time. 

@mi-prakash @ahamed
